src/card.css

```diff
@@ -0,0 +1,87 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
```

Comment:
More correctly to add around padding for card instead of adding left/right margin for each child element. It should be smth like this
![image](https://user-images.githubusercontent.com/54936044/233202025-aad0de68-7b66-4881-922b-5e6cdffbe73f.png)